server/chat-commands/moderation.ts

```diff
@@ -1689,11 +1750,11 @@ exports.commands = {
 		}
 
 		if (user.can('ban')) {
-			const subMap = Punishments.roomIps.get(room.roomid);
+			const sMap = Punishments.roomIps.get(room.roomid);
```

Comment:
Why was this renamed?

Reply:
subMap is already defined in the upper scope.